library/src/scripts/dataLists/dummyEventData.ts

```diff
@@ -0,0 +1,91 @@
+import { EventAttendance } from "@library/events/eventOptions";
```

Comment:
This file is missing license information, and I think it's in the wrong folder. Shouldn't it be in `/events`?